### ISSUE TITLE
Look into gas SDK latency and accuracy [LiFi]

### ISSUE THREAD

**Issue by alexwhte** [connext/monorepo#3685]
From LiFi integration, we have observed 2 issues 

1. often Connext is not showing up in the results 
We believe this is due to latency
we need to have measured latency today as a benchmark
then also look into optimization from changes

2. often our estimates (Arbitreum) are too high, but unclear if this is from our SDK or another improper handling

![image](https://user-images.githubusercontent.com/37959170/223194966-dd87a6c5-c704-4dea-9be5-52ab156a8d3a.png)

**Comment by just-a-node** [comment]
Latency with OP as destination fixed in: #3689 and https://github.com/connext/chaindata/commit/e3935bd67734c41aca91c0ac7632e9e1b5c04f22

**Comment by just-a-node** [comment]
Still need to look into Arbitrum fee estimates, which seem higher than they should be.

**Comment by alexwhte** [comment]
note #3660 for arbitrum fees

**Comment by wanglonghong** [comment]
> Still need to look into Arbitrum fee estimates, which seem higher than they should be.

My deep dive with arbitrum fee for recent transfers, our estimation isn't high than real paid cost. In most cases, it was lower

**Comment by just-a-node** [comment]
I also looked a few transfers today and noticed two patterns:
1) Sometimes the estimate starts initially with a VERY high amount (esp. if origin is Polygon). After, the UI sends estimate requests again and they end up with more reasonable amounts. 
2) Sometimes the estimate is actually lower than the execute fee on destination (??), like Wang mentioned. I saw cases where "LowRelayerFee" was shown but then it would execute shortly after.
